projects/lf-documentation/src/app/lf-field-container-documentation/lf-field-container-demo.service.ts

```diff
@@ -169,39 +187,45 @@ export class LfFieldContainerDemoService implements LfFieldContainerService {
             name: 'Subject',
             id: 11,
             fieldType: FieldType.String,
-            isRequired: true
+            isRequired: true,
+            displayName: "Subject"
```

Comment:
change this to single quote to be consistent

Reply:
this was updated in changelog commit